plugin/db/mysql/rollback.go

```diff
@@ -35,6 +42,96 @@ func (txn BinlogTransaction) GetRollbackSQL(tables map[string][]string) (string,
 	return strings.Join(sqlList, "\n"), nil
 }
 
+// GenerateRollbackSQL generates the rollback SQL statements from the binlog.
+func (driver *Driver) GenerateRollbackSQL(ctx context.Context, binlogFileNameList []string, binlogPosStart, binlogPosEnd int64, tableMap map[string][]string) (string, error) {
```

Comment:
Let's well-define the interface for "GenerateRollbackSQL", as the function name states. What's the input? Otherwise, shall this be an internal function?

1) binlogFileNameStart string, binlogPosStart int64,
2) binlogFileNameEnd string, binlogPosEnd int64,
3) threadID,
4) tableCatalog.

Reply:
Added comments about inputs.